src/main/java/com/onaio/steps/utils/QRCodeUtils.java

```diff
@@ -242,30 +243,22 @@ public static boolean importSettingsFromJSON(AppCompatActivity activity, @NonNul
                 keyValueStore.putString(Constants.IMPORT_URL, householdSettings.getString(Constants.IMPORT_URL));
             }
         } catch (JSONException e) {
-            Log.e(TAG, Log.getStackTraceString(e));
+            Timber.tag(TAG).e(e);
             return false;
         }
 
         return true;
     }
 
     private static void logInfo(String format, String toReplace) {
-        Log.i(TAG, String.format(format, toReplace));
+        Timber.tag(TAG).i(format, toReplace);
```

Comment:
Remove these `Timber.tag(TAG).e/i/w` since we can now do `Timber.e/i/w` and timber will automatically insert the tag

Reply:
done